tests/documentation/DocumentationStyleBearTest.py

```diff
@@ -0,0 +1,22 @@
+import os
+
+from bears.documentation.DocumentationStyleBear import DocumentationStyleBear
+from tests.LocalBearTestHelper import verify_local_bear
+
+
+def load_testfiles(test, good_file, bad_file):
+    good_file_path = os.path.join(os.path.dirname(__file__), "test_files",
+                                  test, good_file)
+    bad_file_path = os.path.join(os.path.dirname(__file__), "test_files",
+                                 test, bad_file)
+    with open(bad_file_path) as bad_f, open(good_file_path) as good_f:
+        return good_f.read(), bad_f.read()
```

Comment:
hehe I've more though about this one: :D 
```python
def load_testfile(name):
    filename = os.path.join(os.path.dirname(__file__), "test_files", "DocumentationStyleBear", name)
    with open(filename) as fl:
        return fl.read()
```

good_file = load_testfile("good_file.py")
bad_file = load_testfile("bad_file.py")
```